src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/root/main.js

```diff
@@ -177,7 +195,7 @@ define(['sulumedia/model/media'], function(Media) {
                 },
                 {
                     el: this.$find(constants.datagridSelector),
-                    url: '/admin/api/media?orderBy=media.changed&orderSort=DESC',
+                    url: '/admin/api/media?orderBy=media.changed&orderSort=DESC&locale=' + this.locale,
```

Comment:
Not related to your PR, but maybe you can fix it also: I think we always write `desc` in lowercase, don't we?